package.json

```diff
@@ -14,7 +14,7 @@
   "license": "MIT",
   "devDependencies": {
     "@twilio-labs/serverless-api": "^1.1.0",
-    "airtable": "^0.8.1",
+    "airtable": "^0.10.1",
```

Comment:
We recently updated the versions of this on the main branch, there is a good chance you need to rebase branch